adafruit_ble/uart.py

```diff
@@ -20,53 +20,104 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
 """
-`adafruit_ble.uuid`
+`adafruit_ble.uart`
 ====================================================
 
 UART-style communication.
 
 * Author(s): Dan Halbert for Adafruit Industries
 
 """
+from bleio import UUID, Characteristic, Service, Peripheral, CharacteristicBuffer
 
-import bleio
-
-from .uuid import UUID
-from .characteristic import Characteristic
-from .service import Service
-from .peripheral_server import PeripheralServer
 
 class UARTService:
```

Comment:
Should this be UARTPeripheral or something else instead of service because it cannot be used with other services?

Reply:
I agree `UARTService` is probably misleading. How about `UARTServer`? It doesn't really act as either a subclass of `Service` or a subclass of `Peripheral`.

Reply:
And it really is officialy a server, in terms of it being a GATT server.